plugins/module_utils/module_container/base.py

```diff
@@ -0,0 +1,1185 @@
+# Copyright (c) 2022 Felix Fontein <felix@fontein.de>
+# Copyright 2016 Red Hat | Ansible
```

Comment:
Why would there be a copyright from 2016 if this was completely rewritten? Or was it not?

Reply:
This contains some code from the old version, so the old copyright needs to stay there AFAIK.

Reply:
> This contains some code from the old version, so the old copyright needs to stay there AFAIK.

That's correct. I'll resolve this.